patches/newlib.patch

```diff
@@ -360,21 +360,21 @@ index 7c23c7a0..2fc58416 100755
          $as_echo "#define HAVE_ELF 1" >>confdefs.h
  
  
-diff --git a/newlib/libc/include/wchar.h b/newlib/libc/include/wchar.h
-index 0d3e636f..cdc3e4ef 100644
---- a/newlib/libc/include/wchar.h
-+++ b/newlib/libc/include/wchar.h
-@@ -69,7 +69,7 @@ typedef __gnuc_va_list va_list;
+diff --git a/newlib/libc/include/sys/features.h b/newlib/libc/include/sys/features.h
+index 218807178..b86caeaff 100644
+--- a/newlib/libc/include/sys/features.h
++++ b/newlib/libc/include/sys/features.h
+@@ -27,6 +27,8 @@ extern "C" {
  
- _BEGIN_STD_C
+ #include <_newlib_version.h>
  
--#if __POSIX_VISIBLE >= 200809 || _XSI_VISIBLE
-+#if 1 // __POSIX_VISIBLE >= 200809 || _XSI_VISIBLE
- /* As in stdio.h, <sys/reent.h> defines __FILE. */
- #if !defined(__FILE_defined)
- typedef __FILE FILE;
++#define _DEFAULT_SOURCE
++
+ /* Macro to test version of GCC.  Returns 0 for non-GCC or too old GCC. */
```

Comment:
This part seems unrelated to the purpose of the patch

Reply:
I think I see what you mean. In the "File Changed" view, it looks like the GCC version test has been added but not if I open the `newlib.patc` file completely  👍